Add env variable to change configuration file path.

victor-rds opened the pull request:
If not set, ETEBASE_INI_PATH defaults to BASE_DIR/etebase-server.ini

As discussed on  #51

tasn commented:
Hey, great, thanks! I have two comments/thoughts:
1. Maybe make the env var in addition to the base path? So if the file in the env var doesn't exist, fallback to the current search paths.
2. Could you maybe rename the env var to `ETEBASE_EASY_CONFIG_PATH` (we call it easy config in the docs if memory serves).

Thanks!

tasn commented:
One more correction (happy to do it myself if you prefer): you changed the order of precedence. The env var settings file should be first, no?

victor-rds commented:
> One more correction (happy to do it myself if you prefer): you changed the order of precedence. The env var settings file should be first, no?

Ok, fixed, I rebased and force pushed, to avoid unnecessary commits

tasn commented:
It crashes when the env var is not set. Should probably only prepend it to the array if it is.

victor-rds commented:
> 
> 
> It crashes when the env var is not set. Should probably only prepend it to the array if it is.

My fault! Was an Null/None error, the initial PR had a default value so the error didn't happened, I just added an empty string default value and everything worked. Sorry :pensive:

tasn commented:
No worries, and thanks!
I'm not making a new release yet, let's wait until we are sure everything works. Once that's the case, I'll make a new release with this patch (and whatever others we may add).

victor-rds commented:
@tasn Sure no problem, I will create a custom Dockerfile for the v0.5.1 using symlinks, but my latest image is always built based on the master branch, and future versions will expect this merge.

tasn commented:
Cool. I can make a release whenever you want me to anyway, so let's just see it works and we don't need anythnig, and I'll make a new one.
